backend/functions/lib/adapters/types/storage.ts

```diff
@@ -1,8 +1,7 @@
 import { CamsRole } from '../../../../../common/src/cams/roles';
-import { DxtrOfficeKeys } from '../gateways/storage/local-storage-gateway';
 
 export type StorageGateway = {
   get(key: string): string | null;
-  getOfficeMapping(): Map<string, DxtrOfficeKeys>;
+  getOfficeMapping(): Map<string, string[]>;
```

Comment:
In `local-storage-gateway`, we replaced the use of `DxtrOfficeKeys` with `GroupDesignators`, so we may want to import and use that in case we ever modify that type to be something other than `string[]`.